<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleSelect</class>
 <widget class="QWidget" name="RoleSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1728</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoleSelecting</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs.qrc">
    <normaloff>:/img/ImagesForCapoo/WindowIcon.png</normaloff>:/img/ImagesForCapoo/WindowIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">RoleSelect
{
background-image: url(:/img/ImagesForCapoo/GameBackGround.jpg);
}</string>
  </property>
  <widget class="QPushButton" name="MuteButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>BackToMenu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModeFighter">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>ModeFighter</string>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/img/ImagesForCapoo/ModeFighter.png</normaloff>:/img/ImagesForCapoo/ModeFighter.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ModeGunner">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>400</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string>ModeGunner</string>
   </property>
   <property name="icon">
    <iconset resource="imgs.qrc">
     <normaloff>:/img/ImagesForCapoo/ModeGunner.jpg</normaloff>:/img/ImagesForCapoo/ModeGunner.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>999</width>
     <height>999</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
